CHANGES.rst

```diff
@@ -96,6 +96,12 @@ ramp_fitting
   of groups in a segment are computed when applying optimal weighting to line
   fit segments. [#7560, spacetelescope/stcal#163]
 
+refpix
+------
+
+- Assign reference pixel flag to first and last four columns for
+  NIRSpec subarrays that do not share an edge with full frame. [#7598]
```

Comment:
```suggestion
  NIRSpec subarrays that do not share an edge with full frame,
  so that corrections can be computed from those unilluminated pixels. [#7598]
```